charts/horizon/values.yaml

```diff
@@ -24,30 +24,15 @@ global:
   # - imagepullsecret1
   # - imagepullsecret2
 
-  image:
-    horizon:
```

Comment:
The horizon image settings are in the global section to allow parent chart to override the value. There is an initial "stellar-network" chart somewhere that we planned to use to deploy full stacks for testing.
I think we should leave it as is

Reply:
a global chart can still overwrite the values that are not in the global section
however, putting values like this in the global section makes the chart less flexible, because any upstream values that might conflict would break a chart

Reply:
> global is a section used to communicate values between charts. The container image horizon should not be overwritten by other charts. 

The intention, from what I remember, was for the horizon image to be overwritten by the parent chart. I'll try to remember some more details.
Could you explain a bit more why you think the container image for horizon should not be overwriten?

> In addition the core exporter image is only used in one place

It's used in 1 place in this chart. But the exporter is also used by the `core` chart